src/components/admin/store/PickupOrdersDisplay/index.tsx

```diff
@@ -0,0 +1,25 @@
+import PickupOrdersPrepareDisplay from '@/components/admin/store/PickupOrdersDisplay/PickupOrdersPrepareDisplay';
+import { Typography } from '@/components/common';
+import { PublicOrderWithItems } from '@/lib/types/apiResponses';
+
+interface PickupOrdersDisplayProps {
+  mode: 'prepare' | 'fulfill';
+  orders: PublicOrderWithItems[];
+}
+
+const PickupOrdersDisplay = ({ mode, orders }: PickupOrdersDisplayProps) => {
+  if (mode === 'prepare') {
+    return <PickupOrdersPrepareDisplay orders={orders} />;
+  }
```

Comment:
I think you should move `mode` and this if statement out of this component and have the page decide whether to use `PickupOrdersDisplay` or `PickupOrdersPrepareDisplay` based on `mode`